cli/cmd/component.go

```diff
@@ -389,6 +427,13 @@ func runComponentsInstall(_ *cobra.Command, args []string) (err error) {
 	cli.OutputChecklist(successIcon, "Component configured\n")
 	cli.OutputHuman("\nInstallation completed.\n")
 
+	params["configure_duration_ms"] = time.Since(start).Milliseconds()
+
+	cli.Event.Component = component_name
+	cli.Event.Feature = "install_component"
```

Comment:
Let's move this assignations (feature and component name) at the beginning of the function, that way if we error out, we will send the Honeyvent with the information already (plus the error information)